app/ports/templates/ports/stats_faq.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'ports/layout.html' %}
+
+{% block title %}FAQ - Statistics |{% endblock %}
+
+{% block content %}
+    <br>
+    <h3>Frequently Asked Questions - Statistics</h3>
+    <hr>
+    <br>
+    <a name="collected-data"></a>
+    <h4>Which Statistics Are Collected?</h4>
+    <hr>
+    <p>The MacPorts statistics collects and submits the following information to the server:</p>
+    <ul>
+        <li>
+            <strong>A unique identifier for your MacPorts installation</strong><br/>
+            This identifier is generated by <code>uuidgen</code> when you first
+            install the statistics service. We use this identifier to overwrite
+            statistics data you submitted earlier – we keep at most one set of
```

Comment:
It is no longer true that we only keep at most one set per month.

Reply:
Thank you for picking this out.